app/models/user.rb

```diff
@@ -138,6 +138,31 @@ def is_admin
     is_global_admin || community_user&.is_admin || false
   end
 
+  # Used by network profile: does this user have a profile on that other comm?
+  def has_profile_on(community_id)
+    cu = community_users.where(community_id: community_id).first
+    !cu&.user_id.nil?
+  end
+
+  def reputation_on(community_id)
+    cu = community_users.where(community_id: community_id).first
+    cu&.reputation
+  end
+
+  def post_count_on(community_id)
+    cu = community_users.where(community_id: community_id).first
+    cu&.post_count
+  end
+
+  def is_moderator_on(community_id)
+    cu = community_users.where(community_id: community_id).first
+    if cu&.is_moderator || cu&.privilege?('mod')
+      true
+    else
+      false
+    end
```

Comment:
`if ... true else false` is redundant and should never be done like that, you can simply return the boolean value directly: `cu&.is_moderator || cu&.privilege?('mod')` (that's it, without the whole construct).